cloud/google/node_init.sh

```diff
@@ -6,4 +6,21 @@ sudo mkdir ${DATA_DIR}
 sudo /usr/share/google/safe_format_and_mount \
   -m "mkfs.ext4 -F" ${DATA_DEV} ${DATA_DIR}
 
-
+# Install google-fluentd which pushes application log files up into the Google
+# Cloud Logs Monitor.
+curl -sSO https://storage.googleapis.com/signals-agents/logging/google-fluentd-install.sh
+sha256sum ./google-fluentd-install.sh
+sudo bash ./google-fluentd-install.sh
+cat > /tmp/ct-info.conf <<EOF
+<source>
+  type tail
+  format none
+  path /data/ctlog/logs/ct-server.*.INFO.*
+  pos_file /data/ctlog/logs/ct-server.INFO.pos
+  read_from_head true
+  tag ct
+</source>
+EOF
+sudo cp /tmp/ct-info.conf /etc/google-fluentd/config.d/ct-info.conf
```

Comment:
You mean it's meant to hook into some init.d style start-up? or a newer 'service foo start' kind of thing?  What / who is going to run it?